Portfolio/index.html

```diff
@@ -38,7 +38,7 @@
     <div class="i">
       GET TO KNOW ME

```

Comment:
The '<br>' tag should not be used in Python code. It is an HTML tag used for line breaks in web pages. Please remove the '<br>' tags from the code.